sources/models/src/modeled_types/shared.rs

```diff
@@ -1,9 +1,9 @@
+use super::error;
```

Comment:
Yeah, I think I had other changes that somehow triggered Clippy warnings, but have since been backed out. Plus I saw some imports were inserted in between the serde Error comment and the actual import. Will drop from here and do a quick little clean up PR.